cookbooks/ros2_windows/attributes/ros2.rb

```diff
@@ -1,3 +1,5 @@
+default['ros2_windows']['ros_distro'] = 'rolling'
```

Comment:
Is there a need for both ros_distro and release_version? Should it be combined into one?

Reply:
Probably not. I'll go through and replace usage of the release_version attribute with ros_distro.